engine/src/main/java/org/hibernate/validator/constraints/SafeHtml.java

```diff
@@ -65,6 +65,13 @@
 	Tag[] additionalTagsWithAttributes() default { };
 
 	/**
+	 * @return Additional whitelist protocols which are allowed on top of the protocols specified by the
+	 * {@link #whitelistType()} for a specified tag's attribute. Adding a protocol will also whitelists the
+	 * given tag and it's attribute.
+	 */
+	Protocol[] additionalProtocols() default { };
```

Comment:
We already got `additionalTagsWithAttributes()`; did you consider to enhance `@SafeHtml.Tag` instead of adding a new annotation?

`@Tag` accepts multiple attributes, so we'd had to add all protocols for all of them (if more than one attribute is specified), but that should be alright.

@oyvindhorneland, WDYT?